loops.js

```diff
@@ -45,9 +61,19 @@
 // >> 9
 // ==========================================
 
-
-
-
+console.log("Opdracht 3:")
+console.log("-------------------------------")
+for (let i = 0; i < 10; i++) {
+    if (i < 3) {
+        console.log(i)
+    } else if (i < 6) {
```

Comment:
"else if" werkt in deze loop prima, let er bij else if alleen wel altijd op dat dit alleen uitvoert als de voorgaande if-statement false is.